core/storage/job/gae_models_test.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Oppia job models."""
+from core.platform import models
+from core.tests import test_utils
+
+entity = "MyEntity"
+(job_models,) = models.Registry.import_models([models.NAMES.job])
+JobModel = job_models.JobModel
+
+
+class JobModelTest(test_utils.GenericTestBase):
+    """Tests for Oppia job models."""
+    def test_get_new_id(self):
+        job_id = JobModel.get_new_id(entity)
+        job = JobModel(id=job_id)
+        self.assertTrue(entity in job_id)
```

Comment:
I'm somewhat confused about what this is testing, exactly. What are you trying to guard against? Every test should have a point (e.g. "initialization happens correctly and matches the args passed in").